DiscordBot/Program.cs

```diff
@@ -51,14 +53,15 @@ private static void ConfigureServices(HostBuilderContext hostContext, IServiceCo
                 .AddScoped<ICatService, CatService>()
                 .AddSingleton<MinecraftService>()
                 .AddTransient<MapBoxStaticMapService>()
-                
-                ;
+                .AddScoped<ICoderDojoAppointmentReaderService, CoderDojoAppointmentReaderService>();
 
             services.Configure<DiscordSettings>(hostContext.Configuration.GetSection("Discord"));
             services.Configure<ImgurSettings>(hostContext.Configuration.GetSection("Imgur"));
             services.Configure<MinecraftSettings>(hostContext.Configuration.GetSection("Minecraft"));
             services.Configure<JawgSettings>(hostContext.Configuration.GetSection("MapServices:Jawg"));
             services.Configure<MapBoxSettings>(hostContext.Configuration.GetSection("MapServices:MapBox"));
+            services.Configure<JawgSettings>(hostContext.Configuration.GetSection("Jawg"));
```

Comment:
Not sure why "Jawg" is now new ..

Reply:
The correct one would be at line #61 : MapServices:Jawg

Reply:
Already removed this with an ammend - should be correct now.